tensorflow/python/kernel_tests/math_ops/reduction_ops_test.py

```diff
@@ -51,6 +51,12 @@ def _powerset(iterable):
   return itertools.chain.from_iterable(
       itertools.combinations(s, r) for r in range(len(s) + 1))
 
+def get_float_types():
+  float_types = [dtypes.float16, dtypes.float32, dtypes.float64]
+  if test_util.is_gpu_available(
+        cuda_only=True, min_cuda_compute_capability=(8, 0)):
+    float_types += [dtypes.bfloat16]
```

Comment:
Is it necessary to run only on Ampere and above? I think bf16 is supported through software emulation on older GPUs as well.